### PR TITLE
Docs: Add fit add and fit delete commands to UG

### PR DESCRIPTION
**Pull request by jjchee77**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -200,6 +200,38 @@ If `/edit` is supplied instead of a note, (e.g. `note 1 /edit`), the contents of
 > Executing the command `note 1 /edit` will replace the contents of the command box with `note 1 Wants to gain muscle`.
 <hr>
 
+### Adding or overriding exercise(s) of clients : `fitadd`
+
+Format: `fitadd INDEX [n/EXERCISE_NAME] [s/SETS] [r/REPS] [b/BREAK_BETWEEN_SETS_IN_SECONDS] [/arms] [/legs] [/chest] [/back] [/shoulders] [/abs] [/all] `
```
Comment (dillontkh) [inline docs/UserGuide.md:205]: `n/EXERCISE_NAME` is not optional, so it should not be in `[ ]`.

Also, should the default exercise prefixes be mentioned separately?

Maybe you could have a subsection about the prefixes specifically.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -200,6 +200,38 @@ If `/edit` is supplied instead of a note, (e.g. `note 1 /edit`), the contents of
...
+
+* Adds the specified exercise(s) to the client specified by `INDEX`. The index refers to the index number shown in the displayed client list. The index **must be a positive integer** 1, 2, 3, …​
+* If an exercise with same name already exists for the client, the old exercise will be overwritten with the newly supplied field values, or a default set of values if not supplied.
```
Comment (dillontkh) [inline docs/UserGuide.md:208]: Might be good to mention the default values.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -200,6 +200,38 @@ If `/edit` is supplied instead of a note, (e.g. `note 1 /edit`), the contents of
...
+
+* `fitadd 1 n/burpees` - Adds or overwrites the `burpees` exercise of the 1st client with a default set of 1, repetition of 1 and 0 seconds break time between sets.
+* `fitadd 1 n/burpees s/3 r/5 b/30` - Adds or overwrites the `burpees` exercise of the 1st client with sets of 3, repetitions of 5 and 30 seconds break time between sets.
+* `fitadd 2 /arms` - Adds or overwrites a default set of exercises from the `arms` category to the 2nd client.
+* `fitadd 2 /arms /legs` - Adds or overwrites a default set of exercises from the `arms` and `legs` category to the 2nd client.
+<hr>
+
+### Deleting exercise(s) of clients : `fitdelete`
+
+Format: `fitdelete INDEX [n/EXERCISE_NAME] [/all]`
```
Comment (dillontkh) [inline docs/UserGuide.md:223]: Same as `fitadd` section

File: docs/UserGuide.md [hunk 4]
```diff
@@ -200,6 +200,38 @@ If `/edit` is supplied instead of a note, (e.g. `note 1 /edit`), the contents of
...
+
+Examples:
+
+* `fitdelete 1 n/burpees` - Deletes the exercise with name `burpees` from the 1st client.
+* `fitdelete 2 /all` - Deletes all exercise(s) from the 2nd client.
+<hr>
+
 ### Searching clients: `find`
 
 Finds all clients that match the specified attributes.
```
Comment (bgopi23) [inline docs/UserGuide.md:237]: image
could  we add fitadd and fitdelete commands to summary section as well?

File: docs/UserGuide.md (hunk 2 above)
Comment (jjchee77) [reply docs/UserGuide.md:208]: Default values are mentioned within the format.

File: docs/UserGuide.md (hunk 1 above)
Comment (jjchee77) [reply docs/UserGuide.md:205]: Technically speaking it is.

You must either specify a specific exercise name or use a number of default supported prefixes, but not both together. Not sure how to represent that.

File: docs/UserGuide.md (hunk 4 above)
Comment (jjchee77) [reply docs/UserGuide.md:237]: Added! Thanks for noticing!

File: docs/UserGuide.md (hunk 3 above)
Comment (jjchee77) [reply docs/UserGuide.md:223]: Technically speaking it is.

You must either specify a specific exercise name or the `/all` prefix, but not both together.
Not sure how to represent that.

File: docs/UserGuide.md (hunk 4 above)
Comment (bgopi23) [reply docs/UserGuide.md:237]: thank u @jjchee77

File: docs/UserGuide.md (hunk 2 above)
Comment (jjchee77) [reply docs/UserGuide.md]: Added!

**Review by dillontkh — APPROVED** [review]
LGTM!

**Pull request merged**